src/main/java/bibliotecame/back/Book/BookController.java

```diff
@@ -43,17 +43,17 @@ public BookController(BookService bookService, TagService tagService, CopyServic
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
         BookModel book = this.bookService.findBookById(id);
-//        if(!book.isActive() && !checkAdmin()){
-//            return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
-//        }
+        if(!book.isActive() && !checkAdmin()){
+            return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+        }
         return new ResponseEntity<>(this.bookService.findBookById(id), HttpStatus.OK);
     }
 
     @PostMapping()
     public ResponseEntity<BookModel> createBook(@Valid @RequestBody BookModel bookModel){
-//        if(checkAdmin()){
-//            return new ResponseEntity<>(bookModel, HttpStatus.UNAUTHORIZED);
-//        }
+        if(checkAdmin()){
```

Comment:
This shouldn't be checking if the user is NOT an admin? 

Note that the same applies to another 4 cases in this same file.